packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -914,12 +914,10 @@ abstract class ResidentHandlers {
     final Brightness? current = await flutterDevices.first!.vmService!.flutterBrightnessOverride(
       isolateId: views.first.uiIsolate!.id!,
     );
-    Brightness next;
-    if (current == Brightness.light) {
-      next = Brightness.dark;
-    } else {
-      next = Brightness.light;
-    }
+    final Brightness next = switch (current) {
+      Brightness.light || null => Brightness.dark,
```

Comment:
My understanding of the if statement is that if and only if `current == Brightness.light` should `next = Brightness.dark`, otherwise (including the `null` case) `next = Brightness.light`. Is this change intended?

Reply:
Good catch, thank you!

The `|| null` should go next to `Brightness.dark`.